src/fswAlgorithms/opticalNavigation/cobConverter/cobConverter.cpp

```diff
@@ -217,7 +221,7 @@ double CobConverter::getRadius() const {
     return this->objectRadius;
 }
 
-/*! Set the attitude error covariance matrix in body frame B
+/*! Set the attitude error covariance matrix in body frame B, for unit vector measurements
```

Comment:
:large_orange_diamond: Should be squashed into the first commit.

Reply:
Done.